From 27e836926abc647871e98e399f1db86e6d53546a Mon Sep 17 00:00:00 2001
From: "Gao,Yan" <ygao@suse.com>
Date: Fri, 6 Jul 2018 14:57:57 +0200
Subject: [PATCH] Fix: stonith:ibmhmc: Add "managedsyspat" and "password" as
 supported parameters

So that the stonith agent correctly receives parameters from environment
variables.

Origin: upstream, https://github.com/ClusterLabs/cluster-glue/commit/27e836926abc647871e98e399f1db86e6d53546a
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1895355
Reviewed-by: Rafael David Tinoco <rafaeldtinoco@ubuntu.com>
Last-Update: 2020-09-15
---
 lib/plugins/stonith/ibmhmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/plugins/stonith/ibmhmc.c b/lib/plugins/stonith/ibmhmc.c
index 5e185364..71ad9456 100644
--- a/lib/plugins/stonith/ibmhmc.c
+++ b/lib/plugins/stonith/ibmhmc.c
@@ -320,7 +320,7 @@ ibmhmc_hostlist(StonithPlugin  *s)
 static const char * const *
 ibmhmc_get_confignames(StonithPlugin* p)
 {
-	static const char * names[] = {ST_IPADDR, NULL};
+	static const char * names[] = {ST_IPADDR, ST_MANSYSPAT, ST_PASSWD, NULL};
 	if (Debug) {
 		LOG(PIL_DEBUG, "%s: called.", __FUNCTION__);
 	}
-- 
2.27.0

